pkl/private/pkl_codegen.bzl

```
@@ -0,0 +1,147 @@
+"""
+Implementation for 'pkl_config_src' and 'pkl_config_java_library' macros.
+"""
+
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+def _to_short_path(f, expander):
```

```suggestion
def _to_short_path(f, _expander):
```

As per the build failure:

```
./pkl/private/pkl_codegen.bzl:7: unused-variable: Variable "expander" is unused. Please remove it. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unused-variable)
```

---

Done